signer/src/transaction_coordinator.rs

```diff
@@ -2548,6 +2548,17 @@ pub async fn should_coordinate_dkg(
     let storage = context.get_storage();
     let config = context.config();
 
+    // Trigger dkg if signer set changes
+    let signer_set_changed = !context
+        .state()
+        .current_signer_set()
+        .has_same_pubkeys(&config.signer.bootstrap_signing_set);
+
+    if signer_set_changed {
```

Comment:
The ticket doesn't mention it, but should we trigger this also if `bootstrap_signatures_required` change?

Reply:
Hmm, why we want to run dkg if signatures required changed? Because if signatures required changed we want to rebuild signers aggregated address changing it's spending condition and we do such changes via dkg?

Reply:
Yes, if the signatures required changed then that implies a different set of secret shares and stuff.

The next question that I have, not for this PR, is whether we want to remove the flexibility of the signer threshold altogether and have it derived from the number of signers in the signing set.